server/views/pages/overview/partials/workAndInterestsTabContents.njk

```diff
@@ -39,7 +39,10 @@
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds">
         <p class="govuk-body">
-          To add work experience and interests information you need to [create a learning and work progress plan] with {{ prisonerSummary.firstName | title }} {{ prisonerSummary.lastName | title }}.
+          To add work experience and interests information you need to
+          {# TODO RR-115 - add link href to CIAG Induction question set #}
```

Comment:
This link needs to take the CIAG back to the CIAG Induction UI (ie. Gary and Mon's service) - so we need to get the base URL from them